# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/compiler/preprocessor/DiagnosticsBase.cpp
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/compiler/preprocessor/DiagnosticsBase.cpp
--- a/gfx/angle/checkout/src/compiler/preprocessor/DiagnosticsBase.cpp	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/compiler/preprocessor/DiagnosticsBase.cpp	Sat Sep 07 13:38:36 2019 +0000
@@ -115,8 +115,8 @@
             return "invalid file number";
         case PP_INVALID_LINE_DIRECTIVE:
             return "invalid line directive";
-        case PP_NON_PP_TOKEN_BEFORE_EXTENSION_ESSL3:
-            return "extension directive must occur before any non-preprocessor tokens in ESSL3";
+        case PP_NON_PP_TOKEN_BEFORE_EXTENSION_ESSL:
+            return "extension directive must occur before any non-preprocessor tokens in ESSL";
         case PP_UNDEFINED_SHIFT:
             return "shift exponent is negative or undefined";
         case PP_TOKENIZER_ERROR:
@@ -129,7 +129,7 @@
             return "unexpected token after conditional expression";
         case PP_UNRECOGNIZED_PRAGMA:
             return "unrecognized pragma";
-        case PP_NON_PP_TOKEN_BEFORE_EXTENSION_ESSL1:
+        case PP_NON_PP_TOKEN_BEFORE_EXTENSION_WEBGL:
             return "extension directive should occur before any non-preprocessor tokens";
         case PP_WARNING_MACRO_NAME_RESERVED:
             return "macro name with a double underscore is reserved - unintented behavior is "